process-compose/suzuka-full-node/process-compose.telemetry.yml

```diff
@@ -0,0 +1,24 @@
+version: "3"
+
+environment:
+
+processes:
+  otlp-collector:
```

Comment:
I don't seem to be getting anything served by this and the e2e tests for the basic simulation are failing. Is this how I should be using this?:

```
just suzuka-full-node native build.setup.eth-local.celestia-local.test.telemetry --keep-project
```

Reply:
<img width="844" alt="Screenshot 2024-10-18 at 10 49 29 AM" src="https://github.com/user-attachments/assets/15d95533-0cac-4c79-8a84-8fdebd45c3d0">

This is all I'm seeing from the collector process when I start with the above.

Reply:
I added a `feed` overlay in case for some reason this is not in fact staying open with the `--keep-project` flag. 

```
just suzuka-full-node native build.setup.eth-local.celestia-local.feed.telemetry --keep-project
```